internal/metrics/metrics.go

```diff
@@ -0,0 +1,75 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+const (
+	CertRestart 		string = "cert_restart"
+	RollingRestart 		string = "rolling_restart"
+	ScheduledRestart 	string = "scheduled_restart"
```

Comment:
@ewolinetz Could you please elaborate which restart case is scheduled only, but neither cert-restart nor rolling-restart?

Reply:
According to what I understood, `scheduled_restart` isn't something different. I guess when `cert_restart` or `rolling_restart` is unsuccessful due to some reason then we schedule them for sometime in future. Is this what it means @ewolinetz ?

Reply:
`cert_restart` means we are scheduled to restart for the certificate.
`scheduled_restart` means it is scheduled to be updated after... i'm not sure what `rolling_restart` is covering here.

We currently just have two different scheduled restart cases

Reply:
@ewolinetz What is your suggestion here? Drop the label value `rolling_restart`? If yes, then the AC is broken too?
```
Expose Operator's current action (e.g. rolling restart, full restart, idle, managed, unmanaged) – Cluster Condition
```

Reply:
I was strictly answering the question asked. I hadn't looked at the AC in relation to what was being done here.
Rolling_restart would be that it currently is in the process of performing a rolling restart.